exonum/src/node/mod.rs

```diff
@@ -850,6 +837,8 @@ pub struct ConnectInfo {
     pub address: String,
     /// Peer public key.
     pub public_key: PublicKey,
+    /// Peer identity key.
+    pub identity_key: Option<PublicKeyKx>,
```

Comment:
Why is `identity_key` optional? What are the scenarios in which `identity_key == None`?